crates/sui-core/src/unit_tests/authority_tests.rs

```diff
@@ -1735,11 +1732,11 @@ async fn test_handle_confirmation_transaction_bad_sequence_number() {
 
     // Explanation: providing an old cert that has already been applied
     //              returns a Ok(_) with info about the new object states.
-    let response = authority_state
+    // TODO: Why was signed_effects None?
```

Comment:
this TODO is not clear, better to add a PR ID for future reference

Reply:
Turns out this test is disabled